apps/angular/src/core/services/anime.service.ts

```diff
@@ -85,3 +83,16 @@ const API_FIELDS = {
   search: 'search',
   typeIn: 'type__in',
 };
+
+/** Params for {getPaginatedAnimeList} method.*/
```

Comment:
```suggestion
/** Params for {@link getPaginatedAnimeList} method.*/
```
https://jsdoc.app/tags-inline-link.html